lldb/source/Plugins/Process/FreeBSD/NativeProcessFreeBSD.cpp

```diff
@@ -48,14 +48,37 @@ static Status EnsureFDFlags(int fd, int flags) {
   return error;
 }
 
+static Status CanTrace() {
+  Status status;
```

Comment:
Suggestion: You can remove this and just return {} below.

Reply:
true